src/screens/HomeScreen.jsx

```diff
@@ -1,10 +1,55 @@
 import React from "react";
-import { View, Text, TouchableOpacity, StyleSheet, Button } from "react-native";
+import {
+  View,
+  Text,
+  TouchableOpacity,
+  StyleSheet,
+  ActivityIndicator,
+  FlatList,
+} from "react-native";
+import { useQuery } from "react-query";
+import { fetchTodo } from "../api-service/todos";
+import { RefreshControl } from "react-native";
 
 const HomeScreen = ({ route, navigation }) => {
+  const {
+    isLoading,
+    data: todos,
+    error,
+    refetch,
+  } = useQuery("myQuery", fetchTodo, { refetchOnWindowFocus: false });
+
+  const renderTodo = ({ item }) => {
+    return (
+      <TouchableOpacity
+        style={styles.todoListItem}
+        onPress={() => navigation.navigate("Details", { item })}
+      >
+        <Text>{item.todo}</Text>
+      </TouchableOpacity>
+    );
+  };
```

Comment:
here is a good way to make component for flatList and also for each item as well